lib/SIL/Utils/OwnershipUtils.cpp

```diff
@@ -339,6 +345,17 @@ bool swift::findExtendedUsesOfSimpleBorrowedValue(
       recordUse(use);
       break;
 
+    case OperandOwnership::GuaranteedForwardingPhi: {
```

Comment:
findExtendedUsesOfSimpleBorrowedValue is just recording the use of Reborrow, and the clients are bailing out if there is a Reborrow before calling this api. I will change findExtendedUsesOfSimpleBorrowedValue to return false when there is a Reborrow (similar to findUsesOfSimpleValue), with that all collected GuranteedForwardingPhi ops will have a dominated borrow root.

Reply:
In `OperandOwnership::GuaranteedForwardingPhi: {` please comment "borrowedValue is known to dominate this phi because if any reborrow exists then this utility returns false

Reply:
Fixed.